src/Cli/Microsoft.DotNet.Cli.Utils/UILanguageOverride.cs

```diff
@@ -97,5 +99,31 @@ private static void SetIfNotAlreadySet(string environmentVariableName, int value
         {
             SetIfNotAlreadySet(environmentVariableName, value.ToString());
         }
+
+        private static bool CurrentWindowsVersionOfficiallySupportsUTF8Encoding()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && Environment.OSVersion.Version.Major >= 10) // UTF-8 is only officially supported on 10+.
```

Comment:
It looks like this judgment is repeated in line 30.

Reply:
This was true in the old code but it has been moved into this function

Reply:
Thanks for taking a look though ;)